components/schema-migrator/validate.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# This script is responsible for validating if migrations scripts are correct.
+# It starts Postgres, executes UP and DOWN migrations.
+# This script requires `compass-schema-migrator` Docker image.
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+INVERTED='\033[7m'
+NC='\033[0m' # No Color
+
+set -e
+
+
+IMG_NAME="compass-schema-migrator"
+NETWORK="migration-test-network"
+POSTGRES_CONTAINER="test-postgres"
+POSTGRES_VERSION="11"
+
+function cleanup() {
+    echo -e "${GREEN}Cleanup Postgres container and network${NC}"
+    docker rm --force ${POSTGRES_CONTAINER}
+    docker network rm ${NETWORK}
+}
+
+trap cleanup EXIT
+
+echo -e "${GREEN}Create network${NC}"
+docker network create --driver bridge ${NETWORK}
+
+echo -e "${GREEN}Start Postgres in detached mode${NC}"
+docker run -d --name ${POSTGRES_CONTAINER} \
```

Comment:
consider adding `--rm` parameter

Reply:
`--rm                             Automatically remove the container when it exits`- but this actually won't happen IMO, because postgres needs to be killed explicitly, which is done cleanup function:
`    docker rm --force ${POSTGRES_CONTAINER}`